hoomd/simulation.py

```diff
@@ -129,15 +129,21 @@ def create_state_from_snapshot(self, snapshot):
         When `timestep` is `None` before calling, `create_state_from_snapshot`
         sets `timestep` to 0.
 
-        Warning:
-            *snapshot* must be a `hoomd.Snapshot`. Use `create_state_from_gsd`
-            to read GSD files. `create_state_from_snapshot` does not support
-            ``gsd.hoomd.Snapshot`` objects from the ``gsd`` Python package.
+        Note:
+            *snapshot* must be either a `hoomd.Snapshot` or a
+            ``gsd.hoomd.Snapshot``. A ``gsd.hoomd.Snapshot`` will first be
```

Comment:
Explicitly document the types accepted in the  `Args:` section - they are more important than a `Note:` block signifies.